src/poker-evaluator.ts

```diff
@@ -20,17 +21,18 @@ export function evalHand(cards: string[] | number[]): EvaluatedHand {
     throw new Error(`Hand must be 3, 5, 6, or 7 cards, but ${cards.length} cards were provided`);
   }
 
-  if (cardsAreValidStrings(cards)) {
+  if (cardsAreValidNumbers(cards)) {
+    if (cards.length === 3) {
+      throw new Error(`Please supply 3 card hands as string[] of "cards" only.`);
+    }
+    return evaluate(cards as number[]);
+  }
+  else if (cardsAreValidStrings(cards)) {
```

Comment:
Small question: why did you invert this if else?

Reply:
It's fine to keep it inverted then! I was just curious and I think the performance gain for the winningOdds() function is worth it, as regular users are very unlikely to call evalHand() w/ strings at the same frequency.